OutOfSchool/OutOfSchool.WebApi/Startup.cs

```diff
@@ -191,7 +192,12 @@ public static async Task AddApplicationServices(this WebApplicationBuilder build
                         Duration = cacheProfilesConfig.PublicDurationInSeconds,
                     });
             })
-            .AddNewtonsoftJson()
+
+            // to handle self-references during serialization
```

Comment:
This comment explain 'what' this code is doing which is bad, but does not explain 'why' this code it is here.